models/gc_tiger_lb2/meta.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "c5397909-0397-489f-8744-6bf3952e9a1c",
+  "name": "tiger_lb2",
+  "title": "TIGER challenge winner: Team VUNO",
+  "summary": {
+    "description": "Participants in the TIGER challenge will have to develop computer algorithms to analyze H&E-stained whole-slide images of breast cancer histopathology, to perform three tasks:  detection of lymphocytes and plasma cells, which are the main types of cells considered as tumor-infiltrating lymphocytes; segmentation of invasive tumor and tumor-associated stroma, which are the main tissue compartments considered when identifying relevant regions for the TILs; compute an automated TILs score, one score per slide, based on the output of detection and segmentation.",
```

Comment:
This is misleading and reads like this is an ongoing challenge. It is not clear that this is about a particular (winning) model from a past challenge.

Reply:
We will update this.